pkgs/by-name/ti/ticktick/package.nix

```diff
@@ -72,7 +80,10 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://ticktick.com/home/";
     license = licenses.unfree;
     maintainers = with maintainers; [ hbjydev ];
```

Comment:
You'll need to add yourself to the maintainers list for ARM, I have no test hardware for the platform I can run NixOS or Nix on.

Reply:
https://github.com/NixOS/nixpkgs/pull/391731

I have personally run this on x86 and arm.